### PR TITLE
Document Updated make:entity command

### PR DESCRIPTION
**Pull request by weaverryan**
Hi guys!

The make:entity command in MakerBundle 1.3 is quite a bit more powerful - it interactively asks for fields (like `doctrine:generate:entity`), but also updating existing entities, supports relationships and generates adders/removers that synchronize the owning side of a relationship.

I've still included what the generated code looks like, because I want users to know the command is optional, and also to see what the code looks like when they're reading.

Cheers!

### CODE REVIEW

File: doctrine.rst [hunk 1]
```diff
@@ -62,14 +63,50 @@ Creating an Entity Class
...
 Suppose you're building an application where products need to be displayed.
 Without even thinking about Doctrine or databases, you already know that
-you need a ``Product`` object to represent those products. Use the ``make:entity``
-command to create this class for you:
+you need a ``Product`` object to represent those products.
+
+.. _doctrine-adding-mapping:
+
+You can use the ``make:entity`` command to create this class and any fields you
+need. The command will ask you some questions - answer them like we've done below:
```
Comment (javiereguiluz) [inline doctrine.rst]: We never use "we" in the docs, so what about: `like we've done below` -> `like done below` ?

**Review by javiereguiluz — APPROVED** [review]
Really nice. I know this command is controversial for some people ... but it will definitely help lots of our readers. Thanks!

**Comment by javiereguiluz** [comment]
Can we please rebase this to merge it? Thanks!

**Comment by weaverryan** [comment]
Fixed! Not with a rebase though... im on my phone... :)

**Comment by xabbuh** [comment]
@weaverryan Something looks really strange here. Why does the PR contain so many changed files?

**Comment by alexislefebvre** [comment]
@xabbuh The merge commit only contains the related changes so it looks good: https://github.com/symfony/symfony-docs/commit/53b231b6d22080fa360a2f898bcb4409ef677d31

**Pull request merged**